src/api/trading/orders.rs

```diff
@@ -1,71 +1,54 @@
 use super::*;
 
-with_builder! { |trading|
-    /// Create an order.
-    #[derive(Serialize, Deserialize, Debug, Clone)]
-    pub struct CreateOrder {
-        /// The symbol/ticker of the stock being traded.
-        pub symbol: String,
-        /// Either the quantity or the dollar amount to trade.
-        #[serde(flatten)]
-        pub amount: OrderAmount,
-        /// Buy or sell.
-        pub side: OrderSide,
-        /// Order type. Includes market, limit, stop, etc. orders.
-        #[serde(flatten)]
-        pub kind: OrderType,
-        // TODO explain
-        pub time_in_force: OrderTif,
-        /// Specifies if the order is allowed to be processed in extended hours.
-        pub extended_hours: bool,
-        // TODO explain
-        #[serde(skip_serializing_if = "Option::is_none")]
-        pub client_order_id: Option<String>,
-        // TODO explain
-        pub order_class: OrderClass
-    }
+/// Create an order.
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct CreateOrder {
+    /// The symbol/ticker of the stock being traded.
+    pub symbol: String,
+    /// Either the quantity or the dollar amount to trade.
+    #[serde(flatten)]
+    pub amount: OrderAmount,
+    /// Buy or sell.
+    pub side: OrderSide,
+    /// Order type. Includes market, limit, stop, etc. orders.
+    #[serde(flatten)]
+    pub kind: OrderType,
+    // TODO explain
+    pub time_in_force: OrderTif,
+    /// Specifies if the order is allowed to be processed in extended hours.
+    pub extended_hours: bool,
+    // TODO explain
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub client_order_id: Option<String>,
+    // TODO explain
+    pub order_class: OrderClass,
 }
 
 impl std::fmt::Display for CreateOrder {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        let Self { symbol, side, amount, kind, time_in_force, extended_hours, order_class, .. } = self;
+        let Self {
+            symbol,
+            side,
+            amount,
+            kind,
+            time_in_force,
+            extended_hours,
+            order_class,
+            ..
+        } = self;
 
-        let allow_ext_hours = extended_hours.then_some(", allowed to process in extended hours").unwrap_or("");
+        let allow_ext_hours = extended_hours
+            .then_some(", allowed to process in extended hours")
+            .unwrap_or("");
 
         // not adding anything after amount because it fmts it nicely in the display impl of
         // OrderAmount
         write!(f, "Creating a {side} {kind:?} order, for {amount} ${symbol} ({order_class}, {time_in_force}{allow_ext_hours})")
```

Comment:
Why is this hardcoded into the SDK? We need an actual, proper logging system.

Reply:
how do you imagine one?
consumers can always log the endpoint structs before they execute them themselves, their own way, and this is just included as a sane-ish out-of-the-box default

Reply:
I agree with the @FilterUnfiltered. We do need a proper logging system.

Reply:
> how do you imagine one? consumers can always log the endpoint structs before they execute them themselves, their own way, and this is just included as a sane-ish out-of-the-box default

that kind of makes sense for the broker api, but when it comes to market data and trading api we want it as accessible as possible. Logging should be built in, easy, and optional, not hardcoded or nonexistent. Sure, they can add a print after every trade to see what their strat is doing, but it would be so much nicer if they could just enable logging for example, and set it to different levels of verbosity ("Minimal", "info", or "debug" for example).

Reply:
This should be left for now. PR is only about migrating existing codebase to Acril.